socialDistribution/app/static/app/js/profile.js

```diff
@@ -135,31 +135,31 @@ function sendFollowRequestToInbox(e){
 
             if (foreign_node_token){
                 headers = new Headers({
-                'Authorization': 'Basic '+foreign_node_token, 
+                'Authorization': 'Basic '+foreign_node_token,
                 'Content-Type': 'application/json'
                 })
             } else {
                 headers = new Headers({
                 'Content-Type': 'application/json'
                 })
             }
-            
+
             fetch(foreignAuthorURL, {
                 method: "POST",
-                headers: headers, 
+                headers: headers,
                 body: JSON.stringify(follow_object)
             }).then(response => {
                 console.log("-------------Response: ", response.status);
             })
-            
+
             const addToSentRequestURL = new URL("add-to-sent", `${window.location.protocol}//` + window.location.host);
-            
+
             const sentRequestObject = {user_id:user_id, author_id:author_id};
 
             fetch(addToSentRequestURL, {
                 method: "POST",
                 headers: new Headers({
-                'Authorization': 'Basic '+btoa('server1:123'), 
+                'Authorization': 'Basic '+btoa('server1:123'),
```

Comment:
These will fail now. as we don't have server1 anymore as a username:password.

Reply:
I think we do have this but this change was only for the space at the end from vs code.